.landscape.yml

```diff
@@ -0,0 +1,2 @@
+python-targets:
```

Comment:
The last run on Landscape was from before I added the classifier to setup.py, works in all the other repos.